### PR TITLE
#543 Add a warning when saving a reaction arrow 

### PR DESCRIPTION
**Pull request by MagomedGasanov**

### CODE REVIEW

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
Please use struct.hasRxnArrow() instead of duplicating this logic here again

**Review by AndreiMazol — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/data/convert/structConverter.ts [hunk 1]
```diff
@@ -26,6 +26,32 @@ export function couldBeSaved(
 ): string | null {
   let warnings: Array<string> = []
   const formatName: string = getPropertiesByFormat(format).name
+
+  const rxnArrowsSize = struct.rxnArrows.size
+  const hasRxnArrow = struct.hasRxnArrow()
+
+  if (format !== 'graph') {
+    if (hasRxnArrow) {
+      const rxnArrowMode = struct.rxnArrows.values().next().value.mode
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/data/convert/structConverter.ts:35]: What do you need it for?

**Review by AndreiMazol — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/data/convert/structConverter.ts (hunk 1 above)
Comment (MagomedGasanov) [reply packages/ketcher-react/src/script/ui/data/convert/structConverter.ts]: Fixed

File: packages/ketcher-react/src/script/ui/data/convert/structConverter.ts [hunk 2]
```diff
@@ -26,6 +27,28 @@ export function couldBeSaved(
 ): string | null {
   let warnings: Array<string> = []
   const formatName: string = getPropertiesByFormat(format).name
+
+  const rxnArrowsSize = struct.rxnArrows.size
+  const hasRxnArrow = struct.hasRxnArrow()
+
+  if (format !== 'graph') {
+    if (hasRxnArrow) {
+      const rxnArrowMode: RxnArrowMode | undefined = struct.rxnArrows.get(0)?.mode
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/data/convert/structConverter.ts:36]: You can use 'struct.rxnArrows.get(0)!' instead (you have already checked that  it exists before - struct.hasRxnArrow() function)

**Review by AndreiMazol — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/data/convert/structConverter.ts (hunk 2 above)
Comment (MagomedGasanov) [reply packages/ketcher-react/src/script/ui/data/convert/structConverter.ts]: Fixed

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**